### DISCUSSION TITLE
"Serialize and copy the state" does not appear to copy to clipboard

### DISCUSSION THREAD

**Discussion by cfjedimaster** [vuejs/pinia#1284] [Help and Questions] [answered]
### Reproduction

This is a bug w/ devtools so I can't share a repo with the issue.

### Steps to reproduce the bug

With devtools open and in an app using Pinia, click on "Serialize and copy the state". Nothing seems to be copied to the clipboard.

### Expected behavior

That it copies to the clipboard.

### Actual behavior

It didn't.

### Additional information

Windows Edge, latest version.

**Comment by posva** [comment, marked as answer]
![Screenshot 2022-05-10 at 08 43 44](https://user-images.githubusercontent.com/664177/167564818-45fc1102-e8a8-4192-95a2-d6a9201cd4de.png)

**Reply by cfjedimaster** [reply]
Ok, that did the trick. Thanks. I had warnings turned off in my devtools and didn't see this.
